### DISCUSSION TITLE
Establishing trials / situations / items

### DISCUSSION THREAD

**Discussion by mariaverde** [olivierfriard/BORIS#343] [Ideas]
I'm a Developmental Psychology PhD candidate from a research team that works with infants and their families. We record infants interacting with their caregivers and experimenters and, afterwards, analyze those interactions.

We are currently starting to analyze our gathered data on the infants' behavior during a task in which an experimenter tries to elicit their imitative responses. We have developed a coding system which we are trying to transfer into Boris to use it as our coding software.

Below is an example of a plot extracted from one of the pilot observations we've done. There are several behaviors coded with a number (1 inicio, 2 pompas, 3 cucutras...). These are not actually behaviors, they are more like items, trials, or situations that occur always throughout the structured imitation task. 

![PLOT_MARIA_V3](https://user-images.githubusercontent.com/75361384/100850120-a6e02680-3483-11eb-94a3-7365fd6edab0.png)

Our suggestion for enhancing Boris would be to create an option for setting these trials (so that they do not appear one below the other on the plot) throughout the recording, so when you actually code the behaviors of the different subjects you are able to know **_when_** or **in which trial** it took place. The ideal would be to see them as a continuum or as columns that temporarily divide the entire graph/plot.

Thank you for considering it!

**Comment by olivierfriard** [comment]
Thank you for your suggestion!
Sorry for the delay...
